examples/mrc/run_mrc.py

```diff
@@ -197,6 +201,12 @@ class DataTrainingArguments:
                     "Otherwise it will be discarded."
         },
     )
+    beam_runner: str = field(
+        default=None,
+        metadata={"help": "The beam runner for loading large dataset.",
+                  "choices": 'DirectRunner',
```

Comment:
Is this needed if there is only one choice right now?

Reply:
changed the choice of beam_runner to be a list.